src/main/java/com/cst438/domain/AssignmentListDTO.java

```diff
@@ -0,0 +1,95 @@
+package com.cst438.domain;
+
+import java.util.ArrayList;
+
+/*
+ * a transfer object that is a list of assignment details
+ */
+public class AssignmentListDTO {
+
+	public static class AssignmentDTO {
+		public int assignmentId;
+		public String assignmentName;
+		public String dueDate;
+		public String courseTitle;
+		public int courseId;
+		
+		public AssignmentDTO() {
```

Comment:
Optional, this doesn't throw an error in Java but could align brace indentation here { }